src/utils/confirmBeforeExit.ts

```diff
@@ -0,0 +1,46 @@
+import { Dialog } from 'quasar';
+import { onBeforeRouteLeave } from 'vue-router';
+
+type ConfirmBeforeExit = {
+  title?: string;
+  message?: string;
+  currentRouteParent: string;
+};
+
+export const confirmBeforeExit = ({
+  title = '¿Estás seguro de que quieres dejar esta página?',
+  message = 'Si sales de la página, se perderán los cambios',
```

Comment:
What about "¿Estás seguro de que quieres abandonar esta página?"

Reply:
Thanks for the suggestion